src/mlpack/methods/ann/rnn.hpp

```diff
@@ -237,26 +237,18 @@ class RNN
    */
   void Shuffle();
 
-  /**
-   * Add a new layer to the model.
-   *
-   * @param layer The layer to be added to the model.
-   */
-  template<typename LayerType>
-  void Add(const LayerType& layer) { network.push_back(new LayerType(layer)); }
-
-  /**
-   * Add a new layer to the model.
+  /*
+   * Add a new module to the model.
    *
-   * @param args The parameters to be passed to the layer constructor.
+   * @param layer The Layer to be added to the model.
```

Comment:
Oops, it looks like the changes make the documentation backwards---I guess this method should have `@param args` and the other should have `@param layer`.